### PR TITLE
Replace ddt usage with parametrize

### PR DESCRIPTION
**Pull request by VeldinaElena**
locally tests work
check_box_test
check_list_test

### CODE REVIEW

File: tests/jdi_uitests_webtests/test/common/check_box_test.py [hunk 1]
```diff
@@ -1,96 +1,70 @@
...
         self.check_box.check()
         CommonActionsData.check_action(MSG_TRUE)
 
-    def test_click(self):
+    def test_click(self, checkbox_setup):
         self.check_box.click()
         CommonActionsData.check_action(MSG_TRUE)
         self.check_box.click()
-        CommonActionsData.check_action(MSG_FALSE)
+        CommonActionsData.check_action(MSG_FALSE)
```
Comment (zhidelev) [inline tests/jdi_uitests_webtests/test/common/check_box_test.py:70]: Add a new line at the end of the file 🙂

File: tests/jdi_uitests_webtests/test/complex/check_list_test.py [hunk 2]
```diff
@@ -1,7 +1,5 @@
-import unittest
-
 import pytest
-from ddt import data, ddt
+
```
Comment (zhidelev) [inline tests/jdi_uitests_webtests/test/complex/check_list_test.py:2]: Please sort imports with `isort`.

File: tests/jdi_uitests_webtests/test/common/check_box_test.py [hunk 3]
```diff
@@ -1,96 +1,70 @@
-import unittest
-
 import pytest
```
Comment (zhidelev) [inline tests/jdi_uitests_webtests/test/common/check_box_test.py:1]: Please sort imports with `isort`.

**Review by zhidelev — CHANGES_REQUESTED** [review]
Good work 👍 
- [ ] Fix comments
- [x] Rename the title of the PR to "Replace ddt usage with parametrize"
- [ ] Squash all commits in one

File: tests/jdi_uitests_webtests/test/common/check_box_test.py (hunk 1 above)
Comment (VeldinaElena) [reply tests/jdi_uitests_webtests/test/common/check_box_test.py:70]: +

File: tests/jdi_uitests_webtests/test/complex/check_list_test.py (hunk 2 above)
Comment (VeldinaElena) [reply tests/jdi_uitests_webtests/test/complex/check_list_test.py:2]: *+*

**Pull request merged**